platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/internal/EventImpl.java

```diff
@@ -997,6 +999,15 @@ public long getMaxRoundCreated() {
         return Math.max(selfParentRound, otherParentRound);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Nullable
```

Comment:
I ask myself if we should add @Nullable to an implementation if the interface already defines it. Any thoughts on that?

Reply:
Do spotless & intellij warn you about null errors if you don't?

Reply:
You can still use `EventImpl` as a type, sans interface.   Since attributes are not first class citizens, I don't think that `EventImpl` inherits the attributes of the interface through the `@Override` attribute.

Reply:
1. https://stackoverflow.com/questions/62435938/should-i-mark-both-the-interface-and-its-implementation-with-the-nonnull-annota
2. https://github.com/projectlombok/lombok/issues/807#issuecomment-538710386